uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -197,6 +198,8 @@ define([
           canSubmit = canSubmit && !_.isEqual(initialLeaveRequestAttributes, this.request.attributes());
           // user has added a comment
           canSubmit = canSubmit || this.request.comments.length !== initialCommentsLength;
```

Comment:
Not sure i'm following fully, but can we fix it in this PR instead of having to open a new ticket? How complicated is it?